package-lock.json

```diff
@@ -1,18 +1,19 @@
 {
     "name": "snbt",
-    "version": "0.0.4",
+    "version": "0.1.1",
     "lockfileVersion": 2,
     "requires": true,
     "packages": {
         "": {
             "name": "snbt",
-            "version": "0.0.4",
+            "version": "0.1.1",
             "dependencies": {
-                "@ironm00n/nbt-ts": "^1.4.0"
+                "@ironm00n/nbt-ts": "^1.4.0",
+                "snbt": "file:"
```

Comment:
Why declare `snbt` itself as one of the dependencies of `snbt`?

Reply:
No reason whatsoever!

I noticed that some extension in my vscode was angry at this project and kept adding `"snbt": "file:"` to the package.json dependencies (only this project, everything else I open is happy). I made sure that was cleared out before building the vsix and testing, but I completely forgot to check the package-lock.json.